text/0000-new-browser-support-policy.md

```diff
@@ -0,0 +1,67 @@
+- Start Date: 2020-05-21
+- Relevant Team(s): All
+- RFC PR: (after opening the RFC PR, update this with a link to it and update the file name)
+- Tracking: (leave this empty)
+
+# New Browser Support Policy
+
+## Summary
+
+> In dropping support for IE11 at some point in the future, we will need a new
+browser support policy. This is the proposed policy.
+
+## Motivation
+
+> It should be clear which browsers are supported by Ember.js
+
+## Detailed policy
```

Comment:
I think it needs to be clear *when* the support matrix is recalculated. I'd like to see this happen only after an LTS release, but I'm not sure quite how to phrase that. 
Maybe:
> Any specific browser version which falls out of support between LTS versions will only be dropped after an LTS supporting it has been released
?

This ensures it's not a constantly recalculated moving target that might shift an app into a non-working state during a minor update cycle.